jest-puppeteer.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   browserContext: 'incognito',
-  dumpio: true,
```

Comment:
I've taken this out as it's supposed to be in `launch` so wasn't working. Can be useful for debugging but adds a lot of noise